### DISCUSSION TITLE
React Components & SliDev

### DISCUSSION THREAD

**Discussion by tutods** [slidevjs/slidev#226] [Q&A]
Is possible using React Components on SliDev? And print like example of Vue component?

If yes anyone can help me?

**Comment by michael-azogu** [comment]
it's built with vue3

**Reply by IPWright83** [reply]
Does that prohibit React completely? I'm wonder if you could wrap a React component with a Vue one? Probably not an amazing approach, but if it enables use in a bunch more projects?
